react-components/src/components/SceneContainer/SceneContainer.tsx

```diff
@@ -31,9 +31,14 @@ export function SceneContainer({
 }: SceneContainerProps): ReactElement {
   const resourceOptions = useReveal3dResourcesFromScene(sceneExternalId, sceneSpaceId);
 
+  const defaultCamera = useSceneDefaultCamera(sceneExternalId, sceneSpaceId);
   useGroundPlaneFromScene(sceneExternalId, sceneSpaceId);
   useSkyboxFromScene(sceneExternalId, sceneSpaceId);
 
+  useEffect(() => {
+    defaultCamera.fitCameraToSceneDefault();
+  }, [defaultCamera]);
```

Comment:
I wonder if this should depend on `sceneExternalId` and `sceneSpaceId` instead, just to make sure it never triggers before the next time we change scene.

Reply:
Fixed in [75d4a73](https://github.com/cognitedata/reveal/pull/4051/commits/75d4a73d1cac7e2ddc85fd5aa8635a3e51bc8acf)